main/pom.xml

```diff
@@ -80,6 +80,12 @@
               <minimizeJar>true</minimizeJar>
               <filters>
                 <!-- Prevent minification from excluding classes looked up by name -->
+                <filter>
+                  <artifact>org.apache.logging.log4j:*</artifact>
```

Comment:
Two blocks down it looks like we have `log4j:log4j`. At some point I guess we switched dependencies and updating this part of the build was missed.

Reply:
yep must have been it!